### ISSUE TITLE
Clean All Icon and Message should be DIFFERENT and STAND OUT

### ISSUE THREAD

**Issue by s7evinz** [Microsoft/vscode#24803]
- VSCode Version: latest
- OS Version: macOS

Steps to Reproduce:

1. wants to revert one file (at the top)
2. accidentally clicks "Clean All" instead because icon and confirm message are basically the same
3. curse endlessly after losing hours of work....

**Comment by joaomoreno** [comment]
The confirmation message is definitely not the same.

![image](https://cloud.githubusercontent.com/assets/22350/25122409/107883b2-2425-11e7-9112-7b64a189ead1.png)

vs

![image](https://cloud.githubusercontent.com/assets/22350/25122410/13b91b72-2425-11e7-8262-383e3c1f88d3.png)

Note the `ALL`, in all-caps.

Sorry that you lost your changes.

We can definitely get a different icon, though.

**Comment by s7evinz** [comment]
Thanks for your response!

Yeah I think having a different icon would help. 

When I was doing it I barely read the message since I've done it so many times and it's just for one file. I just pressed 'enter' to proceed, until I noticed it's taking so long just for "one file" and when I realized what happened it's too late...

It'd be even better if the "Discard Changes" button changes to colour red if you're discarding ALL changes but I don't know how feasible that is.
